### PR TITLE
redefine cert claims based on input from PSA Certified

### LINKED ISSUE #18: Clarification on key PSA Certification aspects from PSA Certification experts
Clarify with PSA Certification Experts:

(a) What are the mandatory and optional elements within a Certificate.

(b) Clarify meaning of Certification Type!

(c) whether the current set is OK (is there any extra or missing claim)

(d) is the data fully public, meaning each and every certified product is advertised on the psacertified.org site?

(e) is there a way to fetch data from the DB?

### LINKED ISSUE #19: add glue CoMID HWVer -> Implementation ID
When the supply chain entity responsible for provisioning certification related data is unable to link it to an Implementation ID, we need a way to make that association (HWVer->ImplID) via an assertion from some other supply chain actor (e.g., the device manufacturer).  This means two things:
1. we need to have the cert data identified by HWVer as an alternative to ImplID (in case the matching ImplID is not known);
2. we need to have a glue CoMID that endorses a specific HWVer as associated to an ImplID.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #18, fixes #19

### CODE REVIEW

File: draft-xyz-rats-psa-endorsements.md [hunk 1]
```diff
@@ -61,28 +61,18 @@ informative:
...
 PSA Endorsements comprise reference values, cryptographic key material and
-certification status information that a Verifier needs in order to appraise PSA
-Evidence.
-
-This memo describes the data formats used to convey PSA Endorsements to an Attestation Verifier.
-The format presented in this document is a PSA specific profile of CoRIM/CoMID based information model presented in
-Trusted Computing Group (TCG) Endorsement Architecture.
+certification status information that a Verifier needs in order to appraise
+attestation Evidence produced by a PSA device.  This memo defines such PSA
+Endorsements as a profile of the CoRIM data model.
```
Comment (yogeshbdeshpande) [inline draft-xyz-rats-psa-endorsements.md:66]: Do we want to refer or clarify more of the CoRIM data model? Where it exists or a link to its reference ?

File: psa-ext/patch.cddl [hunk 2]
```diff
@@ -0,0 +1,13 @@
+comid.psa-fw-patch-triples = 5
+
+$$triples-map-extension //= (
+  comid.psa-fw-patch-triples => one-or-more<psa-fw-patch-triple-record>
+)
+
+security-critical = bool
+
+psa-fw-patch-triple-record = [
```
Comment (yogeshbdeshpande) [inline psa-ext/patch.cddl:9]: Presumably fw-update triple is a TODO, right?

**Review by yogeshbdeshpande — APPROVED** [review]
I do not have major comments on the text. I have made some minor comments.

File: psa-ext/patch.cddl (hunk 2 above)
Comment (thomas-fossati) [reply psa-ext/patch.cddl:9]: I have removed this in 8a6a99e (i.e., blocklists)

File: draft-xyz-rats-psa-endorsements.md (hunk 1 above)
Comment (thomas-fossati) [reply draft-xyz-rats-psa-endorsements.md:66]: L75 has a TBD to be filled as soon as CoRIM gets a reference-able publication

**Pull request merged**